lib/core/src/lib/form/components/widgets/json/json.widget.html

```diff
@@ -0,0 +1,8 @@
+<div class="adf-json-widget">
+    <div class="adf-json-widget-label">
```

Comment:
you can probably go without this div as it does not do much, just use <label> instead

Reply:
I have. followed the same approach as in the text-widget. I have tried removing the div that wraps around the label but it does break the component. We can remove this div but this would require further style rules to the label and button